.travis.yml

```diff
@@ -1,10 +1,5 @@
-language: python
```

Comment:
MQT scripts are python: https://github.com/OCA/maintainer-quality-tools/tree/master/travis

In case of doubts, I would just follow the sample file
https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml